flavors/vpp/vpp_flavor.go

```diff
@@ -32,29 +32,25 @@ func (f *Flavor) Inject() error {
 	}
 	f.injected = true
 
-	f.Base.Inject()
-
-	// Redis plugin
-	//f.Redis.Deps.PluginInfraDeps = *f.Base.FlavorLocal.InfraDeps("redis")
-	//f.RedisSync.Deps.PluginInfraDeps = *f.Base.FlavorLocal.InfraDeps("redis-datasync")
-	//f.RedisSync.KvPlugin = &f.Redis
-	//f.RedisSync.ResyncOrch = &f.Base.ResyncOrch
+	f.EtcdKafka.Inject()
+	f.Redis.Inject()
+	f.RPC.Inject()
 
 	// Aggregated transport
-	adapters := []datasync.KeyProtoValWriter{&f.Base.ETCDDataSync, /*&f.RedisSync*/}
+	adapters := []datasync.KeyProtoValWriter{&f.EtcdKafka.ETCDDataSync, &f.Redis.RedisDataSync}
```

Comment:
write this array directly in following line datasync.CompositeKVProtoWriter{...}